src/backend/gl/src/factory.rs

```diff
@@ -88,6 +88,11 @@ impl Factory {
         }
     }
 
+    /// Get the OpenGL-specific driver information
+    pub fn get_info<'a>(&'a self) -> &'a Info {
```

Comment:
this is highly subjective, but we could just pass `info.shading_language`  to the factory on construction and avoid sharing the `Info` struct